packages/std/src/results/cosmos_msg.rs

```diff
@@ -160,6 +163,23 @@ pub enum WasmMsg {
     ClearAdmin { contract_addr: String },
 }
 
+#[cfg(feature = "stargate")]
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, JsonSchema)]
+#[serde(rename_all = "snake_case")]
+pub enum GovMsg {
+    Vote { proposal_id: u64, vote: Vote },
+}
+
+#[cfg(feature = "stargate")]
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, JsonSchema)]
+#[serde(rename_all = "snake_case")]
+pub enum Vote {
```

Comment:
Let's call this `VoteOption` analogue to Cosmos SDK. There a "Vote" is a vote that was casted.